packages/esm-patient-chart-app/src/patient-chart/action-menu/action-menu.component.tsx

```diff
@@ -22,7 +22,7 @@ export const ActionMenu: React.FC<ActionMenuInterface> = ({ open }) => {
     return () => window.removeEventListener('resize', handleKeyboardVisibilityChange);
   }, [initialHeight]);
 
-  if (open && isTablet) {
+  if (active && workspaceWindowState !== 'hidden' && isTablet) {
```

Comment:
Does this hide the bottom nav on tablet when a workspace is open?

Reply:
Yes @denniskigen!